include/boost/geometry/strategies/cartesian/side_rounded_input.hpp

```diff
@@ -0,0 +1,60 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2021 Tinko Bartels, Berlin, Germany.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGY_CARTESIAN_SIDE_ROUNDED_INPUT_HPP
+#define BOOST_GEOMETRY_STRATEGY_CARTESIAN_SIDE_ROUNDED_INPUT_HPP
+
+#include <boost/geometry/core/config.hpp>
+
+#include <boost/geometry/strategies/side.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace side
+{
+
+
+template <typename CalculationType>
+struct side_rounded_input
+{
+    using cs_tag = cartesian_tag;
+    template <typename P1, typename P2, typename P>
+    static inline int apply(P1 const& p1, P2 const& p2, P const& p)
+    {
+
+        using CT = std::conditional_t
+            <
+                std::is_void<CalculationType>::value,
+                double,
+                CalculationType
+            >;
+
+        using namespace boost::geometry;
+        CT _1 = get<0>(p1);
+        CT _2 = get<1>(p1);
+        CT _3 = get<0>(p2);
+        CT _4 = get<1>(p2);
+        CT _5 = get<0>(p);
+        CT _6 = get<1>(p);
```

Comment:
There are two problems with these names. They are the same as Boost.Bind placeholders. Furthermore typically identifiers starting with `_` are reserved for predefined preprocesor macros. So I propose to change them.

Reply:
One more. They are confusing because these are numbers but they doesnt represents these specific numbers neither a position in some order etc. Better would be e.g. `x` or `p0`, etc.